### ISSUE TITLE
Better Dashboard handling of unexpected domains

### ISSUE THREAD

**Issue by tomjn** [Varying-Vagrant-Vagrants/VVV#2275]
When a domain is loaded that isn't `vvv.test` but isn't handled yet, the dashboard should:

 - if it's vvv.test
    - show the dashboard!
 - check if it's a host in the config file that VVV is meant to serve
    - if so display a message that it's still provisioning, or show error if it failed
 - if it's a raw IP or `vvv`
    - show a prompt telling people to go to vvv.test
 - otherwise
    - show a message

**Comment by Mte90** [comment]
So testing now:

* vvv.test show dashboard
* if ip show an alert

I think that:
```check if it's a host in the config file that VVV is meant to serve ```

it is missing that point but I don't know how to test that case.

**Comment by tomjn** [comment]
I think the best way to implement this is probably via Nginx and static HTML files,  the IP/`vvv` scenario can be handled that way too

**Comment by Mte90** [comment]
So it is something that we need to handle it from VVV and not from the dashboard repo

**Comment by tomjn** [comment]
Indeed,  we currently have code that adds a notice in the  dashboard, but I think a base64 VVV logo on a dark background  with a message in a box and a link to the docs site in a single php file is best. 

Said PHP file would:

 - be self contained
 - each message we would want to show would be its own file
 - not be a part of the dashboard
 - does not live in a synced/mounted folder
 - does not have dependencies
 - is fast and makes no network requests

**Comment by Mte90** [comment]
Seems a plan, so we need to change the labels of the ticket.

**Comment by Mte90** [comment]
with the new nginx default html pages this ticket is still valid?

**Comment by tomjn** [comment]
Yes because if you visit a host that isn't accounted for it still loads the dashboard, the requirements have not been met
